databus-api/src/main/java/com/bazaarvoice/emodb/databus/api/DefaultSubscription.java

```diff
@@ -4,22 +4,51 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.base.Objects;
 
 import java.time.Duration;
 import java.util.Date;
+import java.util.Objects;
+
+import static com.bazaarvoice.emodb.databus.api.Names.isLegalSubscriptionName;
 
 public final class DefaultSubscription implements Subscription {
+    private final static Duration EVENT_TTL_LIMIT = Duration.ofDays(365);
```

Comment:
could we add a comment saying we are choosing 1 year for now, but this is flexible and can be changed if needed.

Reply:
sure, will add comment